src/docs/pages/TimelinePage.tsx

```diff
@@ -0,0 +1,162 @@
+import { FC } from 'react';
+import { CodeExample, DemoPage } from './DemoPage';
+import { Timeline, Button } from '../../lib';
+import { HiArrowNarrowRight, HiCalendar } from 'react-icons/hi';
+
+const TimelinePage: FC = () => {
+  const examples: CodeExample[] = [
+    {
+      title: 'Default timeline',
+      code: (
+        <Timeline>
+          <Timeline.Item>
+            <Timeline.Point>
+              <div className="absolute -left-1.5 mt-1.5 h-3 w-3 rounded-full border border-white bg-gray-200 dark:border-gray-900 dark:bg-gray-700"></div>
```

Comment:
I'm seeing this getting repeated along the usage of the timeline component so I think if it can be moved within the Point component with adding some cutom logic to allow the use to pass an Icon as well.